src/windows/tile_inspector.c

```diff
@@ -708,6 +720,26 @@ static void window_tile_inspector_sort_elements(rct_window *w) {
 	}
 }
 
+static void window_tile_inspector_copy_element(rct_window *w) {
+	// Copy value, in case the element gets moved
+	tileInspectorCopiedElement = *window_tile_inspector_get_selected_element(w);
+	windowTileInspectorElementCopied = true;
+}
+
+static void window_tile_inspector_paste_element(rct_window *w) {
+	rct_map_element *const pastedElement = map_element_insert(windowTileInspectorTileX, windowTileInspectorTileY, tileInspectorCopiedElement.base_height, 0);
```

Comment:
This probably returns `NULL` when the limit has been reached. I've got to test this out, and display an error when that happens.